### ISSUE TITLE
TypeError: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by CMCDragonkai** [rollup/rollup-plugin-babel#161]
I'm using the `transform-runtime` plugin for my library, and I get an error like this:

```
[nix-shell:~/Projects/js-virtualfs]$ npm run rollup

> virtualfs@0.0.1 rollup /home/cmcdragonkai/Projects/js-virtualfs
> rollup --config

🚨   'default' is not exported by node_modules/babel-runtime/core-js/set-immediate.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module                                                    
lib/VirtualFS.js (4:7)
2: import _Set from 'babel-runtime/core-js/set';
3: import _extends from 'babel-runtime/helpers/extends';
4: import _setImmediate from 'babel-runtime/core-js/set-immediate';
          ^
5: /** @module VirtualFS */
```

I have `babel-runtime` in my package.json `dependencies`.

It does work if I remove `resolve` or use:

```
      resolve({
        modulesOnly: true
      }),
```

I'm confused where to report this error, is this from rollup, is it from babel, is it from core-js?

I'm not using the commonjs plugin here so the wiki link doesn't lead me anywhere.

**Comment by CMCDragonkai** [comment]
This has some relationship to: https://github.com/rollup/rollup-plugin-babel/issues/13

In there, the solution was to use the npm plugin which is no the resolve plugin. But here this error occurs when I add in the resolve plugin, it seems maybe the set immediate module in core-js is not standardised.

**Comment by CMCDragonkai** [comment]
With this configuration:

```
import fs from 'fs';
import babel from 'rollup-plugin-babel';
import resolve from 'rollup-plugin-node-resolve';
import commonjs from 'rollup-plugin-commonjs';

const packageJson = JSON.parse(fs.readFileSync('./package.json', 'utf8'));

export default [
  {
    input: 'lib/VirtualFS.js',
    output: {
      file: 'dist/VirtualFS.js',
      format: 'cjs'
    },
    external: Object.keys(packageJson.dependencies),
    plugins: [
      babel({
        babelrc: false,
        // exclude: 'node_modules/**',
        runtimeHelpers: true,
        plugins: ['transform-object-rest-spread', 'transform-runtime'],
        presets: [
          'flow',
          ['env', {
            modules: false,
            targets: {
              node: '6.0.0'
            }
          }]
        ]
      }),
      resolve({
        preferBuiltins: true
      }),
      commonjs()
    ]
  }
];
```

I get this error:

```
[!] TypeError: Cannot convert undefined or null to object
TypeError: Cannot convert undefined or null to object
    at Function.from (native)
    at Identifier.hasEffectsWhenMutated (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:6885:10)
    at MemberExpression.hasEffectsWhenAssigned (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:7148:22)
    at AssignmentExpression.hasEffects (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:5979:51)
    at included.someChild.child (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:5699:58)
    at keys.some.key (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:5786:11)
    at Array.some (native)
    at ExpressionStatement.someChild (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:5779:20)
    at ExpressionStatement.hasEffects (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:5699:32)
    at ExpressionStatement.shouldBeIncluded (/home/cmcdragonkai/Projects/js-virtualfs/node_modules/rollup/dist/rollup.js:5775:15)
```

**Comment by CMCDragonkai** [comment]
This is now fixed with the latest rollup version 0.51.3.
